docs/deploying-airbyte/deploying-airbyte.md

```diff
@@ -0,0 +1,182 @@
+
+  # Database Secrets
+  database-host: ## e.g. database.internla
+  database-port: ## e.g. 5432
+  database-name: ## e.g. airbyte
+  database-user: ## e.g. airbyte
+  database-password: ## e.g. password
+
+  # Instance Admin
+  instance-admin-email: ## e.g. admin@company.example
+  instance-admin-password: ## e.g. password
+
+  # SSO OIDC Credentials
+  client-id: ## e.g. e83bbc57-1991-417f-8203-3affb47636cf
+  client-secret: ## e.g. wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+
+  # AWS S3 Secrets
+  s3-access-key-id: ## e.g. AKIAIOSFODNN7EXAMPLE
+  s3-secret-access-key: ## e.g. wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+
+  # AWS Secret Manager
+  aws-secret-manager-access-key-id: ## e.g. AKIAIOSFODNN7EXAMPLE
+  aws-secret-manager-secret-access-key: ## e.g. wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+
+```
+
+**Option 2: Create the secret in the CLI**
+
+You can also use `kubectl` to create the secret directly from the CLI:
+
+```sh
+kubectl create secret generic airbyte-config-secrets \
+  --from-literal=license-key='' \
+  --from-literal=database-host='' \
+  --from-literal=database-port='' \
+  --from-literal=database-name='' \
+  --from-literal=database-user='' \
+  --from-literal=database-password='' \
+  --from-literal=instance-admin-email='' \
+  --from-literal=instance-admin-password='' \
+  --from-literal=s3-access-key-id='' \
+  --from-literal=s3-secret-access-key='' \
+  --from-literal=aws-secret-manager-access-key-id='' \
+  --from-literal=aws-secret-manager-secret-access-key='' \
+  --namespace airbyte
+```
+
+
+</TabItem>
+<TabItem value="GCS" label="GCS">
+
+First, create a new file `gcp.json` containing the credentials JSON blob for the service account you are looking to assume.
+
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: airbyte-config-secrets
+type: Opaque
+stringData:
+  # Enterprise License Key
+  license-key: ## e.g. xxxxx.yyyyy.zzzzz
+
+  # Database Secrets
+  database-host: ## e.g. database.internal
+  database-port: ## e.g. 5432
+  database-name: ## e.g. airbyte
+  database-user: ## e.g. airbyte
+  database-password: ## e.g. password
+
+  # Instance Admin Credentials
+  instance-admin-email: ## e.g. admin@company.example
+  instance-admin-password: ## e.g. password
+
+  # SSO OIDC Credentials
+  client-id: ## e.g. e83bbc57-1991-417f-8203-3affb47636cf
+  client-secret: ## e.g. wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+
+  # GCP Secrets
+  gcp.json: <CREDENTIALS_JSON_BLOB>
+```
+
+Using `kubectl` to create the secret directly from the `gcp.json` file:
+
+```sh
+kubectl create secret generic airbyte-config-secrets \
+  --from-literal=license-key='' \
+  --from-literal=database-host='' \
+  --from-literal=database-port='' \
+  --from-literal=database-name='' \
+  --from-literal=database-user='' \
+  --from-literal=database-password='' \
+  --from-literal=instance-admin-email='' \
+  --from-literal=instance-admin-password='' \
+  --from-file=gcp.json
+  --namespace airbyte
+```
+
+</TabItem>
+</Tabs>
+
```

Comment:
The content is good. I'd suggest a relatively large overhaul of the structure here. If I'm sitting down to set up Airbyte, I would find myself navigating up and down this document a lot. I think we should make it a more step-by-step process (kinda like you already did with the quickstart).

1. set up your secrets
2. set up your ingress
3. set up your infrastructure (links to your infrastructure pages? not entirely clear what's going in there, depending on what it is, it might be the first step or an optional step at the end.)
4. "At this point you are ready to run Airbyte. Run `helm install` ... OR if you'd like to configure integrations you can do these optional steps <links to the other integrations>"

Reply:
Moved things around and added some more tutorial like information on exactly what steps you will need to take to actually run a helm install (also now part of that pages doc). 

Going to work on AWS Infra docs tomorrow.